src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFloorAdhesionOldObject.java

```diff
@@ -0,0 +1,99 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+public class AdsFloorAdhesionOldObject extends AdsBaseObject {
+
+  private static final String FLOOR_ADHESION_CSS = "#ext-wikia-adEngine-template-floor";
+  private static final String FLOOR_ADHESION_AD_FRAME_CSS =
+      "#ext-wikia-adEngine-template-floor .ad iframe";
+  private static final String FLOOR_ADHESION_IMAGE_IN_FRAME_CSS = "img";
+  private static final String FLOOR_ADHESION_CLOSE_CSS = "#ext-wikia-adEngine-template-floor .close";
+  private static final String WIKIA_BAR_CSS = "#WikiaBar";
+
+  public AdsFloorAdhesionOldObject(WebDriver driver, String testedPage) {
+    super(driver);
+    getUrl(testedPage, true);
+  }
+
+  public void verifyFloorAdhesionPresent(String expectedSlotName, String expectedLineItemId,
+                                         String expectedCreativeId) {
+    verifyGptAdInSlot(expectedSlotName, expectedLineItemId, expectedCreativeId);
+    wait.forElementVisible(By.cssSelector(FLOOR_ADHESION_CSS));
+    PageObjectLogging.log(
+        "Check visibility of Floor Adhesion",
+        "Floor Adhesion should be displayed",
+        true
+    );
+  }
+
+  public void verifyThereIsNoFloorAdhesion() {
+    waitForElementNotVisibleByElement(driver.findElement(By.cssSelector(FLOOR_ADHESION_CSS)));
+    PageObjectLogging.log(
+        "Check visibility",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Check visibility" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)